resources/assets/components/pages/VoterRegistrationDrivePage/Alpha/AlphaPage.js

```diff
@@ -39,7 +39,9 @@ const AlphaPage = ({ userId }) => {
               ? 'Urge your friend to vote based on the causes you care about most.'
               : null
           }
-          shareCardTitle={betaPageEnabled ? 'Share with your friends' : null}
+          shareCardTitle={
+            betaPageEnabled ? 'Share with your friends' : 'Your online drive'
+          }
```

Comment:
Unless you think this is _worse_, maybe worth using `undefined` to force the default prop, so we can avoid redefining that value here?
```suggestion
          shareCardTitle={ betaPageEnabled ? 'Share with your friends' : undefined }
```

Reply:
It looks like the test still fails. This feature won't be on for longer than this week, so I think it'll be safe to change it back to providing the duplicate default value for now.

Reply:
I think it fails because the capitalization is inconsistent between the test and the default prop, but agreed that it's no biggie for the short term in any case!